xAPI-Communication.md

```diff
@@ -1276,45 +1283,45 @@ Stores, changes, fetches, or deletes the specified profile document in the conte
 **Content (GET | DELETE):** None.  
 
 **Returns (PUT | POST | DELETE)** `204 No Content`  
-**Returns (GET):** `200 OK`, the Profile document  
+**Returns (GET):** `200 OK`, the profile document  
 <table>
 	<tr><th>Parameter</th><th>Type</th><th>Description</th><th>Required</th></tr>
 	<tr id="2.7.s3.table1.row1">
 		<td>activityId</td>
-		<td>Activity Id (IRI)</td>
-		<td>The Activity Id associated with this profile.</td>
+		<td>Activity id (IRI)</td>
+		<td>The Activity id associated with this profile.</td>
 		<td>Required</td>
 	</tr>
 	<tr id="2.7.s3.table1.row2">
 		<td>profileId</td>
 		<td>String</td>
-		<td>The profile Id associated with this profile.</td>
+		<td>The profile id associated with this profile.</td>
 		<td>Required</td>
 	</tr>
 </table>
 
 ###### <a name="2.7.s4"></a>Multiple Document GET
 Example resource endpoint: http://example.com/xAPI/activities/profile
 
-Fetches Profile Ids of all profile entries for an Activity. If "since" parameter is specified, this is limited to 
+Fetches profile ids of all profile entries for an Activity. If "since" parameter is specified, this is limited to 
 entries that have been stored or updated since the specified Timestamp (exclusive).  
 
 **Content:** None.
 
-**Returns:** `200 OK`, Array of Profile Ids  
+**Returns:** `200 OK`, Array of profile ids  
```

Comment:
See comment above about statement id(s). There could also be 1 profile id returned here.